clang/lib/CodeGen/Targets/RISCV.cpp

```diff
@@ -632,6 +633,29 @@ class RISCVTargetCodeGenInfo : public CommonCheriTargetCodeGenInfo {
 
   void setTargetAttributes(const Decl *D, llvm::GlobalValue *GV,
                            CodeGen::CodeGenModule &CGM) const override {
+    const auto *VD = dyn_cast_or_null<VarDecl>(D);
+    if (VD) {
```

Comment:
You can combine these two lines: `if (const auto *VD = dyn_cast<VarDecl>(D)) {`

Reply:
https://github.com/CHERIoT-Platform/llvm-project/pull/147/commits/730e14ac9ac441112cb249c017943195245f0123#diff-2412b0407edd59fe07f0e2547e92c7e79fe352bb491447240b96d4272e62e035R636 combines them in a single line.